USAGE

```diff
@@ -0,0 +1,54 @@
+toTemplate_flow pipeline
+=====================
+
+Register t1, metrics and tractograms to template space.
+We suggest to use https://github.com/scilus/combine_flows to create input.
+
+
+USAGE
+
+nextflow run main.nf [OPTIONAL_ARGUMENTS] --input --template
+
+DESCRIPTION
+
+    --input=/path/to/[root]     Root folder containing multiple subjects
+                                                [root]
+                                                ├── S1
+                                                │   │── *t1.nii.gz (diff space)
+                                                |   └── metrics (optional)
+                                                |   │   └── *.nii.gz
+                                                |   └── tractograms (optional)
+                                                |       └── *.trks
+                                                └── S2
+                                                    └── *
+
+    --template                                  Template used for registration.
```

Comment:
I guess a T1 is best, but are there other images that would work? Should we write that a T1 is expected, or at least that a nifti file is expected?